plugins/modules/template_workflow_manager.py

```diff
@@ -1466,7 +1483,7 @@ def get_containing_templates(self, containing_templates):
 
             name = item.get("name")
             if name is None:
-                self.msg = "name is mandatory under containing templates"
+                self.msg = "name is required under containing templates"
```

Comment:
The parameter 'name' is required under 'containing_templates' but not provided..